### ISSUE TITLE
Feature request: set a password

### ISSUE THREAD

**Issue by vinivosh** [Killianoni/1fichier_debrid#1]
What it says on the tin.

It'd be great to be able to set a password to be used in all links in `links.txt`. Or maybe even accept a format of `<url> | <password for this url>` or something like that

**Comment by Killianoni** [comment]
Good idea i'll Work on it tbh i leaved the project but u gave me good motivation i'll keep u updated ty !
